### PR TITLE
clarified registry instructions for docker >=1.12

### PR DESCRIPTION
**Pull request by koryk**
### Describe the proposed changes
Updated documentation to reflect this http://stackoverflow.com/questions/38695515/can-not-pull-push-images-after-update-docker-to-1-12

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview processing.

Built with commit 8e1d64f29caa5152a39e6168bd98a2a59549a94b

https://app.netlify.com/sites/docker-docs/deploys/582216e8a7e2d212767593f8

File: registry/insecure.md [hunk 1]
```diff
@@ -26,7 +26,15 @@ environment.
 
     Depending on your operating system, your Engine daemon start options.
 
-2. Edit (or add) the `DOCKER_OPTS` line and add the `--insecure-registry` flag.
+2. **If using docker version 1.12 or greater**
+
+    Create or modify /etc/docker/daemon.json on the client machine and add:
```
Comment (johndmulhausen) [inline registry/insecure.md]: Wrap the path in backticks, please :)

File: registry/insecure.md [hunk 2]
```diff
@@ -26,7 +26,15 @@ environment.
...
+     
+    `{ "insecure-registries":["myregistry.example.com:5000"] }`
+
```
Comment (johndmulhausen) [inline registry/insecure.md]: Would look better if this line were encased in a backtick fence using three backticks, like so:

```
{ "insecure-registries":["myregistry.example.com:5000"] }
```

Wrapping it in single backticks is just for inline use. I think you're trying to show a snippet here.

**Comment by johndmulhausen** [comment]
Close! But I think you want the triple backticks to have their own line:

```
1. ```
2. code
3. ```
```

It looks crunched together and weird right now if you check out this PR's deploy preview: 
https://deploy-preview-492--docker-docs.netlify.com/registry/insecure/

**Comment by joaofnfernandes** [comment]
@thaJeztah Can you confirm that we've changed the configuration file, or this is not version-dependent but OS-dependent instead?

**Comment by thaJeztah** [comment]
@joaofnfernandes this didn't change in 1.12. the `/etc/default/docker` file is only used on distro's that don't use systemd (e.g. Ubuntu 12.04 or Ubuntu 14.04), not sure on what platforms the `/etc/sysconfig/docker` is used. For platforms running systemd (just about everything else), these files are not used.

I think we should remove the whole mention of the `/etc/sysconfig/docker`, and `/etc/default/docker`, and only mention the `daemon.json` approach; using `daemon.json` works on every distro, and does not depend on the init-system in use, so even works if you manually start the daemon from the command line

**Comment by joaofnfernandes** [comment]
@thaJeztah totally agree on a platform-agnostic way of configuring Docker. @koryk are you up for changing your PR 💪 to reflect this or should we close this PR and file an issue 😢 to keep track of this feedback?

**Comment by mstanleyjones** [comment]
Ping @koryk @joaofnfernandes what's the status of this PR?

**Comment by joaofnfernandes** [comment]
According to @thaJeztah this PR does not reflect the behavior of the product. I've asked @koryk to update the PR to reflect what is discussed on this thread, but haven't heard back in a couple of days, so feel free to close it.

**Comment by joaofnfernandes** [comment]
I'm closing this PR since it doesn't reflect the product behavior, and opened issue #1116 so that we can track this.

**Pull request closed without merging**